specification/billing/resource-manager/Microsoft.Billing/privatepreview/2020-11-01-privatepreview/billing.json

```diff
@@ -0,0 +1,361 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-11-01-privatepreview",
+    "title": "BillingManagementClient",
+    "description": "Billing client provides access to billing resources and logs for Azure subscriptions."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Billing/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Lists the available billing REST API operations.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingActivityList": {
+      "get": {
+        "tags": [
+          "BillingActivity"
+        ],
+        "x-ms-examples": {
+          "BillingActivityList": {
+            "$ref": "./examples/BillingActivityList.json"
+          }
+        },
+        "operationId": "BillingActivityList_Get",
+        "description": "Gets billing activity logs for a billing account",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/startDateParameter"
+          },
+          {
+            "$ref": "#/parameters/endDateParameter"
+          },
+          {
+            "$ref": "#/parameters/activityLogEventCategoryParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BillingActivityList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "BillingActivityList": {
```

Comment:
Is there any reason we're not following the same activity log model as here? https://docs.microsoft.com/en-us/rest/api/monitor/tenantactivitylogs/list

Reply:
1. This api exposes audit logs captured on the Ea portal and is different from the Azure activity logs.
2. Ea audit logs are captured at billing account scope.